spec/models/api/commodity_spec.rb

```diff
@@ -10,12 +10,6 @@
     allow(Uktt::Commodity).to receive(:new).and_call_original
   end
 
-  describe '.resource_key' do
```

Comment:
This is no longer required since we pass positional config to the retrieve method and the base resource and id is the name of the resource id